gcp-go-gke/.gitignore

```diff
@@ -0,0 +1,353 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
This gitignore appears to be for csharp. Can we scrap this given that this is a go example?